### DISCUSSION TITLE
Adapting Config file for use with a Becker Hickl Time Tagger PCI card

### DISCUSSION THREAD

**Discussion by rdoherty51** [Ulm-IQO/qudi-iqo-modules#119] [Q&A] [answered]
Hello all,

I hope you are well. I have been working to set up Qudi for use in a confocal microscope used to detect fluorescence from NV colour centres. I have installed Qudi and it is running with the default configuration. I am afraid that coding is not my specialty and I was wondering if you could help me change the config file to recognise our time tagger. We are using an SPC-130IN time tagger from Becker Hickl connected to an Excelitas SPCM-AQRH-14-FC single photon detector. I think I understand the general structure of the config files but I am not sure what changes I need to make. Would you be able to provide me with instructions on how to integrate this into Qudi?

Many thanks in advance.

**Comment by Geegee** [comment, marked as answer]
Sadly at least we at IQO don't have this hardware to my knowledge,
hence you would need to implement your own hardware file for your SPC-130IN time tagger. Depending on what you plan
to do I would suggest looking at the fastcounter interface
and hardware files implementing this interface. For general structure of qudi I would guide you to the paper
https://www.sciencedirect.com/science/article/pii/S2352711017300055 .
Also helpful would be https://github.com/Ulm-IQO/qudi-core/blob/main/docs/getting_started.md especially the
part about "New qudi measurement modules".
In any case feel free to ask further questions.
